docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       SQLALCHEMY_DATABASE_URI: "postgresql+psycopg2://user:password@postgres:5432/default_db"
       CAMUNDA_API_URL: "http://host.docker.internal:8081/engine-rest"
       PLUGIN_RUNNER_URLS: "http://host.docker.internal:5005"
+      LOCALHOST_PROXY_PORTS: ":9090"
```

Comment:
The plugin runner also needs to access the registry (and the camunda workflow tool).

Reply:
These two are already configured in other env vars to be access via host.docker.internal. Or should I change them to localhost and add the ports to the proxy config?

Reply:
For the registry using host.docker.internal should not cause any problems, but with camunda, we may end up accessing it with both URLs/passing URLs to the UI. I think it is better to be fully consistent to avoid confusion. So everything (besides the databases) should be localhost.